### PR TITLE
Inclusive language statement for MTA 5.1.1+ documents

### PR DESCRIPTION
**Pull request by RichardHoch**
Resolves https://issues.redhat.com/browse/WINDUP-2969.
Preview: http://file.emea.redhat.com/rhoch/25012021/inc_web_c/html-single/#making-open-source-more-inclusive_web-console-guide

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick: please review.

**Comment by RichardHoch** [comment]
@apinnick: all MTA 5.1.1 documents updated.

File: docs/cli-guide_5/master.adoc [hunk 1]
```diff
@@ -12,6 +12,9 @@ include::topics_5/templates/document-attributes.adoc[]
 
 = {UserCLIBookName}
 
+//Inclusive language statement
+include::topics_5/making-open-source-more-inclusive.adoc[leveloffset=+1]
```
Comment (apinnick) [inline docs/cli-guide_5/master.adoc:16]: The RHEL docs do not specify a [level offset](https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/rhel-8-docs/-/blob/master/enterprise/titles/planning/preparing-for-disaster-recovery-with-identity-management/master.adoc). 

Please remove the `leveloffset=+1` from each `include::` in all assemblies and then I will merge.

Comment (RichardHoch) [reply docs/cli-guide_5/master.adoc:16]: I looked in their assembly files and they do specify the level offset. It's +1 for all includes.

**Comment by RichardHoch** [comment]
@apinnick; level offsets removed.

File: docs/plugin-guide_5/master.adoc [hunk 2]
```diff
@@ -84,4 +87,4 @@ include::topics_5/templates/revision-info.adoc[]
 // Workaround for PDF generation build failure
 {nbsp}
```
Comment (apinnick) [inline docs/plugin-guide_5/master.adoc:88]: What is `{nbsp} for?

Comment (apinnick) [reply docs/plugin-guide_5/master.adoc:88]: I see it was added in 2017. You can delete lines 87 and 88.

**Pull request merged**